### DISCUSSION TITLE
Ability to disable custom text editor

### DISCUSSION THREAD

**Discussion by ahknight** [home-assistant/frontend#8389] [Accessibility]
The custom text editor (YAML, etc.) is pretty horrible outside of Chrome in its current state.  It eats key commands, doesn't tab correctly, doesn't copy/paste correctly, and is very nearly unusable on iOS.  It's also forced on me with the Companion apps and renders them impossible to use for anything more than turning things on and off.  It makes editing exceedingly frustrating for non-Chrome users in general, and on iOS where there is no choice of using Chrome's rendering or JS engine it means that whole sections of the interface are inaccessible.

I'd like a way to just disable it and have a plain text field until the editor works on all major browsers.

**Comment by spacegaier** [comment]
FYI: There is a migration to a newer code editor version underway. That might fix many of the mentioned issues: https://github.com/home-assistant/frontend/pull/8382
